packages/core/dialect/saphana/debug.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (ns){
```

Comment:
Is this file mandatory? If not, please, consider removing

Reply:
Also, please remove that DS_store file and add it to .gitignore on project root

Reply:
no actually this file is left over before move to package manager